Source/Core/Common/MemArena.cpp

```diff
@@ -63,7 +63,7 @@ void MemArena::GrabSHMSegment(size_t size)
 #else
 	for (int i = 0; i < 10000; i++)
 	{
-		std::string file_name = StringFromFormat("dolphinmem.%d", i);
+		std::string file_name = StringFromFormat("/dolphinmem.%d", i);
```

Comment:
Read the manpage. Every Linux and OS X say to start with a slash for it to be portable, except FreeBSD which appears to mandate it.